noir/noir-repo/test_programs/execution_success/bigint/src/main.nr

```diff
@@ -26,3 +26,37 @@ fn big_int_example(x: u8, y: u8) {
     println(d[0]);
 }
 // docs:end:big_int_example
+
+
+
+fn test_u256(mut x: [u8; 5], y: [u8; 5]) {
```

Comment:
We can put these test functions inside of the stdlib.

This is also not being run currently.

Reply:
The function is now called by the test.
I have not added it to stdlib test because I do not see any other test in the stdlib

Reply:
https://github.com/noir-lang/noir/blob/d202a086790f6dae33f9df385ab39b64ec8b54db/noir_stdlib/src/field/bn254.nr#L105-L181